test/runnable/testscope.d

```diff
@@ -294,7 +294,7 @@ void test11()
     static int i;
     bar11(p, &i);
 
-    bar11((i,p), &i);  // comma expressions are deprecated, but need to test them
+    bar11(p, &i);
```

Comment:
This test was specifically testing the comma operator.  Now that it produces an error, it should probably be moved to the fail_compilation tests.  However, given that the equivalent expression already exists on line 295, it would probably be ok, to just delete it.

Reply:
Yeah, I was very careful not to delete anything, but yeah in this case it does make sense. Thanks!